gen.py

```diff
@@ -47,6 +48,7 @@ class PlotFlags(OutFlags):
     template: FilePath = Field(None, description="Default plot template")
 
 
+
 class DepModel(BaseModel):
```

Comment:
Not sure about the new lines around classes in general. Otherwise LGTM. @skshetry ?